spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -0,0 +1,256 @@
+      Code biases are to be added to pseudorange.
+      The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - code:
+            type: u8
+            desc: Signal constellation, band and code
+            fields:
+              - 0-7:
+                  values:
+                      - 0: GPS L1CA
+                      - 1: GPS L2CM
+                      - 2: SBAS L1CA
+                      - 3: GLO L1CA
+                      - 4: GLO L2CA
+                      - 5: GPS L1P
+                      - 6: GPS L2P
+     - integer_indicator:
+            type: u8
+            desc: Indicator for integer property
+     - widelane_integer_indicator:
+            type: u8
+            desc: Indicator for two groups of Wide-Lane(s) integer property
+     - discontinuity_counter:
+            type: u8
+            desc: |
+              Signal phase discontinuity counter. 
+              Increased for every discontinuity in phase.
+     - bias:
+            type: s32
+            units: 0.1 mm
+            desc: Phase bias for specified signal
+ - PhaseBiasesPackedContent:
+     short_desc: SSR phase biases corrections for a particular satellite.
+     desc: |
+      Code biases are to be added to pseudorange.
+      The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+     - yaw:
+            type: u16
+            units: 1 / 256 semi-circle
+            desc: Satellite yaw angle
+     - yaw_rate:
+            type: s16
+            units: 1 / 8192 semi-circle / s
+            desc: Satellite yaw angle rate
+     - biases:
+            type: array
+            fill: PhaseBiasesValues
+            map_by: code
+            desc: |
+              Phase biases corrections for a
+              satellite being tracked.
+
+ - MSG_SSR_ORBIT_CLOCK:
+    id: 0x005DC
+    short_desc: Precise orbit and clock correction
+    desc: |
+      The precise orbit and clock correction message is 
+      to be applied as a delta correction to broadcast 
+      ephemeris and is typically an equivalent to the 1060
+      and 1066 RTCM message types
+    fields:
+        - header:
+            type: OrbitClockHeader
+            desc: Header of a precise orbit/clock correction message
+        - correction:
+            type: array
+            fill: OrbitClockPackedContent
+            map_by: sid
+            desc: |
+              Orbit and clock corrections for a
+              satellite being tracked.
+              
+ - MSG_SSR_CODE_BIASES:
+    id: 0x005E1
+    short_desc: Precise code biases correction
+    desc: |
+      The precise code biases message is to be added
+      to the pseudorange of the corresponding signal
+      to get corrected pseudorange. It is typically 
+      an equivalent to the 1059 and 1065 RTCM message types
+    fields:
+        - header:
+            type: OrbitClockHeader
```

Comment:
Code biases are sharing the OrbitClockHeader?

Reply:
Yes, the only difference in the headers in RTCM is the satellite reference datum field, which is proposed to be dropped anyway sbp orbit/clock message header